capture-sdk/sdk/src/main/java/net/gini/android/capture/internal/util/AndroidHelper.java

```diff
@@ -12,6 +12,13 @@
  */
 public final class AndroidHelper {
 
+    //We need this only while app is running
+    //Shared preferences would be maybe overthinking
+    //Let me know if we don't want to use this
+
+    public static int STORE_SCROLL_STATE = -1;
+    public static int STORE_DOCUMENTS_SIZE = -1;
```

Comment:
[Suggestion] I would propose to not store any global state. It would make it tough to rediscover later how state is managed and it also couples state management of the review screen to other classes (for example the CameraActivity). Also debugging can be more difficult with global states.

Instead I would propose to do the following:
* Camera screen knows when an image was added and tells the review screen whether to scroll to the last page or not: the `MultiPageReviewActivity.createIntent()` receives a boolean parameter called `shouldScrollToLastPage` and forwards this to `MultiPageReviewFragment.createInstance()` where it can be passed to the fragment via an attribute.
* Remember which page was shown when returning to the review screen: using `STORE_SCROLL_STATE` is ok for now. We will have to [change navigation so that the review screens becomes root after taking a picture](https://ginis.atlassian.net/browse/PIA-3146) so I am fairly sure we will be able to drop `STORE_SCROLL_STATE` then.